packages/story-editor/src/components/library/panes/text/textSets/textSet.js

```diff
@@ -113,9 +115,12 @@ function TextSet({ id, elements, translateY, translateX, ...rest }, ref) {
     [onClick]
   );
 
-  const renderImages =
-    typeof WEB_STORIES_DISABLE_OPTIMIZED_RENDERING !== 'undefined' &&
-    WEB_STORIES_DISABLE_OPTIMIZED_RENDERING !== 'true';
```

Comment:
Ignore this change, this is reverting a PR that causes multiple test failures, see https://github.com/GoogleForCreators/web-stories-wp/pull/10589#issuecomment-1042875858